homeassistant/const.py

```diff
@@ -468,6 +468,10 @@
 SPEED_KILOMETERS_PER_HOUR = f"{LENGTH_KILOMETERS}/{TIME_HOURS}"
 SPEED_MILES_PER_HOUR = "mph"
 
+# Signal_strength units
+SIGNAL_STRENGTH_DECIBELS = "dB"
+SIGNAL_STRENGTH_DECIBELS_MILLIWATT = "dBm"
```

Comment:
```suggestion
SIGNAL_STRENGTH_DECIBELS_MILLIWATT = f"{SIGNAL_STRENGTH_DECIBELS}m"
```

Reply:
So, why are we using an f-string? I think it would make more sense to add the actual unit here...

Reply:
IMHO, both work fine.
Is just a matter to choice a way and then be consistent across all.

Simone

Reply:
It would just be consistent with the rest of units because they use f-strings as well.

Reply:
Some needs review as well like 

- SPEED_MILES_PER_HOUR = "mph" => f"mp{TIME_HOURS}"
- DATA_KILOBITS = "kbit" > f"k{DATA_BITS}"

Simone

Reply:
>  just be consistent with the rest of units because they use f-strings as well

I'm not considering "because it has been done before" a valid reason.

Basically we are using constants now... but actually are not using them (if that makes sense in a way?)

If the goal is to provide conversions in the future, I think this is bad practice.

Reply:
Once this PR is committed, I'll take care of creating a new one to make the whole UNITs consistent to the final decision.
Just let me know which is ;-)

Simone

Reply:
@frenck you re probably rightnow I agree with Frenck 😅 your explanation makes more sense and I think it is much easier to look up the meaning of a constant by not using f Strings.

Reply:
@frenck can you expand your reasoning around conversion? I don't follow that.